src/Avalonia.Visuals/Media/Color.cs

```diff
@@ -273,8 +290,15 @@ public uint ToUint32()
         }
 
         /// <summary>
-        /// Check if two colors are equal.
+        /// Returns the HSV color model equivalent of this RGB color.
         /// </summary>
+        /// <returns>The HSV equivalent color.</returns>
+        public HsvColor ToHsv()
```

Comment:
I added this helper conversion method to Color. Now the build is broken. Why HsvColor is unavailable to Color in certain situations is beyond me. This seems like a poorly configured project? Color and HsvColor are now completely the same scope, namespace and project. This should just work.

Reply:
I had to manually add HsvColor to `src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj` to fix this. I'm not sure why it's setup that way, seems if build tasks needs these types it would just pull in the whole project or use globbing.

Reply:
Since .net does not have static linking we have to copy files to the compiler to remove dependency on Avalonia projects. I'm not sure if globbing can work in this case since we are only pulling in a specific part of dependencies (in certain cases only a part of the file).

Reply:
Alright, thanks for the details. I think this could have been configured a bit better but I won't bother with it further here in this PR.